tools/update_runnablejobs.py

```diff
@@ -243,6 +226,49 @@ def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=Fa
                           .values(buildsystem=_buildsystem)
             conn.execute(statement)
 
+
+def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=False):
+    if not new_data:
+        return
+
+    # This is a loop to remove duplicates (including buildsystem -> * if needed)
+    # By doing this it allows us to have a single database query
+    # no_duplicates will contain a subset which excludes:
+    #   * jobs that don't specify the build_platform
+    #   * jobs that don't specify the testtype
+    #   * XXX: unclear why we append the job again but with *
```

Comment:
we do not append the job with *, instead we replace the buildsystem type with '*' if we find an identical job between both systems.